app/presenters/idv/address_presenter.rb

```diff
@@ -0,0 +1,31 @@
+module Idv
+  class AddressPresenter
+    def initialize(pii:)
+      @pii = pii
+    end
+
+    def puerto_rico_address?
+      @pii[:state] == 'PR'
+    end
+
+    def pii
+      @pii
+    end
+
+    def address_line1_hint
+      I18n.t('forms.example') + ' 150 Calle A Apt 3' if puerto_rico_address?
```

Comment:
I'm guessing that Rubocop will say the line is too short for the block form of `if`. And no, we don'd need safe_join... unless `t('forms.example')` ever contains HTML, which it might one day? 🤷 

in the meantime this might be a good compromise for readability 

```suggestion
      "#{I18n.t('forms.example')} 150 Calle A Apt 3" if puerto_rico_address?
```

Reply:
Good idea, did that.